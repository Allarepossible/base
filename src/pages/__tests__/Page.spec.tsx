import React from 'react';
import {shallow} from 'enzyme';

import Page from '../Page';

it('Renders Page', () => {
    const result = shallow(
        <Page title="title">Content</Page>);

    expect(result).toMatchSnapshot();
});
