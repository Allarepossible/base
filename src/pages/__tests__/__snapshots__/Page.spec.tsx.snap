// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Page 1`] = `
<styled.div>
  <styled.div>
    <h1>
      title
    </h1>
    Content
  </styled.div>
</styled.div>
`;
